thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertTaskRunner.java

```diff
@@ -139,7 +145,24 @@ private void runTask() throws Exception {
     }
 
     sendAlertForAnomalies(collectionAlias, results, groupedResults, dimensionNames);
-    updateNotifiedStatus(results);
+    updateNotifiedStatus(allResults);
+  }
+
+  private List<MergedAnomalyResultDTO> applyFiltrationRule(List<MergedAnomalyResultDTO> results) {
+    if (results.size() == 0) {
+      return results;
+    }
+    // TODO: configure filtration rule at metric level and fetch it from DB
+    AnomalyFiltrationHelper.FiltrationRule filtrationRule =
```

Comment:
For a given MergedAnomalyResultDTO, multiple rules can be applied. Want to make sure that  you can get a list of filtration Rules, based on metric_id and/or pattern type.  For simplicity let's do metric first, but we can have at least more than 1 rules.